### PR TITLE
Windows Defender ATP: Feature - Add action to update a machines tag

### PR DESCRIPTION
**Pull request by dfederschmidt**
#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [x] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [x] If this is a new connector or you are adding new actions
    - [x] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [x] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [x] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [x] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [x] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
- Adds a new action to add/remove machine tags from devices

### CODE REVIEW

**Review by dfederschmidt — COMMENTED** [review]
Normally, this should be valid syntax from Python 3.6 onwards.

File: windowsdefenderatp_connector.py [hunk 1]
```diff
@@ -783,9 +783,9 @@ def _handle_test_connectivity(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         self.save_progress(DEFENDERATP_MAKING_CONNECTIVITY_MSG)
 
-        self.save_progress(f"Login URL: {self._login_url}")
-        self.save_progress(f"Graph URL: {self._graph_url}")
-        self.save_progress(f"Resource URL: {self._resource_url}")
+        self.save_progress("Login URL: {}".format(self._login_url))
```
Comment (dfederschmidt) [inline windowsdefenderatp_connector.py:786]: Normally, this should be valid syntax from Python3.6 onwards.

Comment (mishalp-crest) [reply windowsdefenderatp_connector.py:786]: Agreed!
But also wanted to keep things consistent

Comment (dfederschmidt) [reply windowsdefenderatp_connector.py:786]: Understood - that makes sense, consistency is good :) Thanks!

**Review by ricardo-phantom — APPROVED** [review]

**Pull request merged**